1ejer/index.css

```
@@ -0,0 +1,235 @@
+.ejer1-container{
+    max-width: 1200px;
+    margin: auto;
+    display: flex;
+    gap: 24px;
+    padding: 20px;
+}
+
+.left{      
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+}
+
+.title{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 54px;
+    text-align: center;
+    color: #000000;
+    margin-top: auto;
+}
+
+.reference {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+    line-height: 1.33;
+    letter-spacing: normal;
+    font-weight: normal;
+    text-align: center;
+    color: #000000;
+}
+
+.description {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    line-height: 1.25;
+    letter-spacing: normal;
+    text-align: center;
+    color: #000000;
+}
+
+.ejer1-container button {
+    width: 260px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 18px;
+    letter-spacing: 1px;
+    text-align: center;
+    color: #ffffff;
+    margin: 24px auto 42px auto;
+    padding: 15px 16px;
+    background-color: #fb6a66;
+    border:none;
+}
+
+.ejer1-container button:active {
+    background-color: #fb6faa;
+    cursor: pointer;
+}
+
+.ejer1-container button:hover {
+    cursor: pointer;
+}
+
+.image-container {
+      width: 50%;
+      margin-top: 42px;
+      margin-bottom: 42px;
+      padding-top: 30px;
+      padding-bottom: 20px;
+}
+
+.ejer1-container .image-container img{
+      max-width: 688px;
+      width: 100%;
+      margin-left: 0px;
+      margin-right: auto;
+      margin-top: 42px;
+      margin-bottom: 42px;
+      padding: 20px 0 20px 0;
+}
+
+.container2 {
+    max-width: 1200px;
+    margin: auto;
+    display: flex;
+    gap: 24px;
+    padding: 20px;
+}
+
+.container2-rigth {
+    width: 30%;
+    display: flex;
+    flex-direction: column;
+}
+
+.image-container2 {
+    width: 70%;
+}
+
+.container2 .image-container2 img {
+      max-width: 584px;
+      width: 100%;
+}
+
+.h3 {
+    font-family: PlayfairDisplay;
```

Para que estas fuentes funcionen deberías agregar en el HTML

`<link href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,600,700,800|Playfair+Display:400,700|Roboto:300,400,500,70&display=swap" rel="stylesheet"/>`

y cambiar ésto a 

```suggestion
    font-family: 'Playfair Display';
```

---

El "link" va en la parte del <head>